src/engine/Settings.cs

```diff
@@ -111,7 +111,14 @@ public enum StrainBarVisibility
     ///   enabled.
     /// </summary>
     [JsonProperty]
-    public SettingValue<float> MicrobeDistortionStrength { get; private set; } = new(0);
+    public SettingValue<float> MicrobeDistortionStrength { get; private set; } = new(0.6f);
```

Comment:
As there's a new default value, there should be `[JsonProperty(PropertyName = "MicrobeDistortionStrengthV2")]` on this property to ensure all players get the new default value.